.gitignore

```diff
@@ -3,15 +3,3 @@
 # If you find yourself ignoring temporary files generated by your text editor
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile '~/.gitignore_global'
-
-# Ignore bundler config.
-/.bundle
```

Comment:
I think you do want this .bundle in your git ignore file - was there a specific reason you deleted it?